prow/cluster/monitoring/mixins/prometheus/tide_alerts.libsonnet

```diff
@@ -0,0 +1,55 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'Tide progress',
+        rules: [
+          {
+            alert: 'TideNoProgressKK',
+            expr: |||
+              clamp_min(
+              (sum(merges_sum{org="kubernetes",repo="kubernetes",branch="master"}) or vector(0))
+               - (sum(merges_sum{org="kubernetes",repo="kubernetes",branch="master"} offset 5m ) or vector(0)),
+              0) < 0.5
+              and
+              max(min_over_time(pooledprs{org="kubernetes",repo="kubernetes",branch="master"}[4h])) > 0.5
```

Comment:
@stevekuznetsov 

Just tried the same query with `org="openshift",repo="origin"` ... our tide seems fine in the past month.
I would suggest to bring this cool query out of my prom. knowledge back to our production.

Reply:
> I would suggest to bring this cool query out of my prom. knowledge back to our production.

Is this a request for me? If so, please clarify.

Reply:
No, was a comment to me